x/bridge/keeper/assets.go

```diff
@@ -0,0 +1,60 @@
+package keeper
+
+import (
+	"fmt"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v23/x/bridge/types"
+)
+
+type ChangeAssetStatusResult struct {
+	OldStatus types.AssetStatus
+	NewStatus types.AssetStatus
+}
+
+func (k Keeper) ChangeAssetStatus(
```

Comment:
Lets add a general godoc here, specifically what we mean by assetStatus and any contracts (i.e., asset must exist to be called, etc).

Reply:
Done